mtgjson5/set_builder.py

```diff
@@ -1038,3 +1040,71 @@ def get_base_and_total_set_sizes(set_code: str) -> Tuple[int, int]:
     total_set_size = len(ScryfallProvider().download_cards(set_code))
 
     return base_set_size, total_set_size
+
+
+def add_tcgplayer_details(mtgjson_set: MtgjsonSetObject) -> MtgjsonSetObject:
+    """
+    Adds a list of all sku ids from TCGPlayer
+    :param mtgjson_set: MTGJSON Set
+    """
+
+    LOGGER.info(f"Adding TCGPlayer details for {mtgjson_set.code}")
+    list_of_product_ids: List
+    group_sku_data: Dict[str, List]
+    condition_map = {"1": "NM", "2": "LP", "3": "MP", "4": "HP", "5": "DMG", "6": "U"}
+    language_map = {
+        "1": "EN",
+        "2": "CS",
+        "3": "CT",
+        "4": "FR",
+        "5": "GE",
+        "6": "IT",
+        "7": "JP",
+        "8": "KR",
+        "9": "PT",
+        "10": "RU",
+        "11": "SP",
+    }
+    for mtgjson_card in mtgjson_set.cards:
```

Comment:
I feel this entire for loop can be replaced with a list comprehension... maybe something like

```python
list_of_product_ids = [mtgjson_card.identifiers.tcgplayer_product_id for mtgjson_card in mtgjson_set.cards if mtgjson_card.identifiers.tcgplayer_product_id]
```